vocabulary.md

```diff
@@ -1,30 +1,34 @@
+- **Conversational Application** - a software application that runs on a Dialogue System and has been programmed to achieve a bounded set of tasks.   For the purposes of the Open Voice Network, Conversational Application is synonymous with the term Voice Application. 
 - **Conversational Platform** - the system or services that handle the two-way interaction with the user and the entity.  The platform owns the task of understanding the user's utterance, translating it to humans, and machine exchange, and making sure it is reaching the right entity. 
 - **Component Interoperability** - A component of a platform may be replaced by another component from a different vendor.  I.e., replace the TTS by another TTS from a different vendor.
 
 ## D
 - **Dialog Broker (DB)** - is responsible for providing the fulfillable intents available for a resolved VRS record (e.g. where resolved VRS record "BigGrocery", it’s fulfillable intents might be "order product, check order status, add to shopping list". These fulfillable intents can execute remotely on the DM or download locally on the device.
 - **Dialog Manager (DM)** - handles the dynamic response of the conversation. It provides a more personalized response based on the action provided by the NLP to send back to the user.
+- **Dialogue System** - is an Agent that communicates with another Agent in natural language (text, speech, or both).  Speech Dialogue Systems are minimally comprised of an automated speech recognizer (ASR), a Dialog Manager, and speech output which may include pre-recorded audio or text-to-speech (TTS).  Examples of Dialogue Systems include the platforms that deliver VoiceXML, IVR's, Google Assistant, IBM Watson. 
 - **Disambiguate** - when the conversational platform hypothesizes two or more possible resolutions to a user utterance, it may ask the user for additional clarification or choose between the various interpretations to decide the user's correct intention. 
 
 
 ## E
-- **Entity** - is a part of the structured machine translation. It is a custom level data type and considered a concrete value to associate a word in a query. This is also known as annotations.
-- **Entity** -  government or private/public company that provides certain businesses or services.
+- **Entity** - the OVN uses this term in two ways.  From a technology-data perspective, is a part of the structured machine translation. It is a custom level data type and considered a concrete value to associate a word in a query. This is also known as annotations. 
+- **Entity** - the OVN uses this term in two ways.  From a business-systems perspective, it describes a person, organization, or enterprise that engages with a Dialogue System to achieve a desired outcome.  
```

Comment:
Looks like you included two `Entity` lines here.

Reply:
Dan, thanks.  In our conversation right before the Holiday break, we had noted that we were using entity in two different ways -- and, after discussion and the addition of some clarifying words,  that doing so would be OK.    Please see the clarifying language -- "from a technology-data perspective" and "from a business-systems perspective"  Will this work?